File: examples/js_vs_python/class.js
class Sequence {
  constructor(header, seq) {
    this.header = header;
    this.seq = seq;
  }

  reverse() {
    return new Sequence(
      this.header,
      this.seq.split("").reverse().join("")
    );
  }
}

const seq = new Sequence("test", "ACGTA");
seq.reverse();
// Object { header: "test", seq: "ATGCA" }

File: examples/js_vs_python/class.py
class Sequence:
    def __init__(self, header, seq):
        self.header = header
        self.seq = seq

    def reverse(self):
        return Sequence(
            self.header,
            self.seq[::-1]
        )


seq = Sequence("test", "ACGTA")
seq.reverse()
# <__main__.Sequence object at 0x1011f5750>
